# **Scribe**: Towards inferring causal regulations with single cell dynamics-coupled measurements

Single-cell transcriptome sequencing now routinely samples thousands of cells, potentially providing enough data to reconstruct *causal gene regulatory networks* from observational data. Here, we developed **Scribe**, a toolkit for detecting and visualizing causal regulations, and explore the potential for single-cell experiments to power network reconstruction. **Scribe** employs *Restricted Directed Information* to determine causality by estimating the strength of information transferred from a potential regulator to its downstream target by taking advantage of time-delays. We apply **Scribe** and other leading approaches for network reconstruction to several types of single-cell measurements and show that there is a dramatic drop in performance for "pseudotime” ordered single-cell data compared to live imaging data. We demonstrate that performing causal inference requires temporal coupling between measurements. We show that methods such as “*RNA velocity*” restore some degree of coupling through an analysis of chromaffin cell fate commitment. These analyses therefore highlight an important shortcoming in experimental and computational methods for analyzing gene regulation at single-cell resolution and point the way towards overcoming it. 
